src/components/widgets/Number.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
+
+import Header from './Header'
+
+import "../../styles/widget/number.css"
+import "../../styles/widget/widget.css"
+
+export default class Number extends Component {
+  calculateProgress() {
+    const { statistic, goal } = this.props
+    return ( statistic / goal ) * 100
+  }
+
+  roundTwoDecimals( data ) {
+    return parseFloat( data ).toFixed( 2 )
+  }
+
+  calculateProgressPercentage() {
+    return this.roundTwoDecimals( this.calculateProgress() )
+  }
+
+  render() {
+    const { title, x, y, statistic, goal } = this.props
+    const positionStyle = { top: x, left: y }
+
+    const progress = this.calculateProgressPercentage()
+
+    const isComplete = statistic / goal >= 1
+
+    return (
```

Comment:
Oh and make the presentation component (Number) a functional component. Example:
` const Number = ( props ) => (<article>...</article>)`